apps/astarte_realm_management_api/lib/astarte_realm_management_api_web/views/error_view.ex

```diff
@@ -39,6 +39,33 @@ defmodule Astarte.RealmManagement.APIWeb.ErrorView do
     %{errors: %{detail: "Forbidden"}}
   end
 
+  def render("missing_token.json", _assigns) do
+    %{errors: %{detail: "Missing authorization token"}}
+  end
+
+  def render("invalid_token.json", _assigns) do
+    %{errors: %{detail: "Invalid JWT token"}}
+  end
+
+  # TODO: add error message for invalid_token_signature
+
+  def render("invalid_auth_path.json", _assigns) do
+    %{
+      errors: %{
+        detail:
+          "Authorization failed due to an invalid path. Ensure the realm name and endpoint are correctly specified in the request"
```

Comment:
Nit: the path does not contain an Astarte endpoint, since this is Realm Management API. See https://docs.astarte-platform.org/astarte/latest/api/?urls.primaryName=Realm%20Management%20API for more